public/app/features/alerting/unified/components/contact-points/ContactPoints.v2.tsx

```diff
@@ -1,5 +1,267 @@
+          <ContactPointHeader name={'New school'} provenance={'api'} />
+          <div className={styles.receiversWrapper}>
+            <Stack direction="column" gap={0}>
+              <ContactPointReceiver type={'slack'} description="#test-alerts" sendingResolved={false} />
+              <ContactPointReceiver type={'discord'} />
+            </Stack>
+          </div>
+        </Stack>
+      </div>
+
+      <div className={styles.contactPointWrapper}>
+        <Stack direction="column" gap={0}>
+          <ContactPointHeader name={'Japan 🇯🇵'} />
+          <div className={styles.receiversWrapper}>
+            <ContactPointReceiver type={'line'} />
+          </div>
+        </Stack>
+      </div>
+
+      <div className={styles.contactPointWrapper}>
+        <Stack direction="column" gap={0}>
+          <ContactPointHeader name={'Google Stuff'} />
+          <div className={styles.receiversWrapper}>
+            <ContactPointReceiver type={'googlechat'} />
+          </div>
+        </Stack>
+      </div>
+
+      <div className={styles.contactPointWrapper}>
+        <Stack direction="column" gap={0}>
+          <ContactPointHeader name={'Chinese Contact Points'} />
+          <div className={styles.receiversWrapper}>
+            <Stack direction="column" gap={0}>
+              <ContactPointReceiver type={'dingding'} />
+              <ContactPointReceiver type={'wecom'} error="403 unauthorized" />
+            </Stack>
+          </div>
+        </Stack>
+      </div>
+
+      <div className={styles.contactPointWrapper}>
+        <Stack direction="column" gap={0}>
+          <ContactPointHeader
+            name={
+              "This is a very long title to check if we are dealing with it appropriately, it shouldn't cause any layout issues"
+            }
+          />
+          <div className={styles.receiversWrapper}>
+            <Stack direction="column" gap={0}>
+              <ContactPointReceiver type={'dingding'} />
+            </Stack>
+          </div>
+        </Stack>
+      </div>
+    </Stack>
+  );
+};
+
+interface ContactPointHeaderProps {
+  name: string;
+  provenance?: string;
+  policies?: string[]; // some array of policies that refer to this contact point
+}
+
+const ContactPointHeader = (props: ContactPointHeaderProps) => {
+  const { name, provenance, policies = [] } = props;
+
+  const styles = useStyles2(getStyles);
+  const isProvisioned = Boolean(provenance);
+
+  return (
+    <div className={styles.headerWrapper}>
+      <Stack direction="row" alignItems="center" gap={1}>
+        <Stack alignItems="center" gap={1}>
+          <Span variant="body">{name}</Span>
+        </Stack>
+        {policies.length > 0 ? (
+          <MetaText>
+            {/* TODO make this a link to the notification policies page with the filter applied */}
+            is used by <Strong>{policies.length}</Strong> notification policies
+          </MetaText>
+        ) : (
+          <MetaText>is not used</MetaText>
```

Comment:
I think the information that a contact point is not used should be eye-catching. It can mean a misconfiguration of your system

Reply:
Yea I agree – maybe a badge like the one we have for provisioning could work 🤔

Reply:
I'll follow-up on this in a separate PR :)